fpdf/fpdf.py

```diff
@@ -247,11 +249,13 @@ def __init__(
         self.page = 0  # current page number
         self.pages = {}  # array of PDFPage objects starting at index 1
         self.fonts = {}  # map font string keys to an instance of CoreFont or TTFFont
-        self.images = {}  # map image identifiers to dicts describing the raster images
-        self.icc_profiles = {}  # map icc profiles (bytes) to their index (number)
         self.links = {}  # array of Destination objects starting at index 1
         self.embedded_files = []  # array of PDFEmbeddedFile
 
+        # ImageCacheI:
```

Comment:
This should really be encapsulated in its own instance (see my other comment).

Reply:
Alright. I'm going to answer your other comment